recipes/pinocchio/meta.yaml

```diff
@@ -0,0 +1,54 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```

Comment:
Please remove the comments in line 1-5 and 14-15

Reply:
Thanks for the feed-back and sorry I forgot about the comments.

Note I marked the PR WIP as an attempt to indicate I was not sollicitating reviews for now. I edited the top post to make this more clear.

Reply:
Of course @jan-janssen if you have a better way make the WIP status of this PR more imediate this for potential reviewers, I'd be happy to hear about it!

Note that it is extremely useful to have the CIs (this definitely surfaced a few problems/limitations in the pinocchio cmake files) so the draft PR github functionality (from what I have heard CI does not run on a draft PR) ...